src/os_win/os_mtx_cond.c

```diff
@@ -37,11 +37,13 @@ __wt_cond_alloc(WT_SESSION_IMPL *session,
 }
 
 /*
- * __wt_cond_wait --
- *	Wait on a mutex, optionally timing out.
+ * __wt_cond_wait_signal --
+ *	Wait on a mutex, optionally timing out.  If we get it
+ *	before the time out period expires, let the caller know.
  */
 int
-__wt_cond_wait(WT_SESSION_IMPL *session, WT_CONDVAR *cond, uint64_t usecs)
+__wt_cond_wait_isngal(WT_SESSION_IMPL *session,
```

Comment:
Fix the function spelling: `__wt_cond_wait_signal` instead of `__wt_cond_wait_isngal`.